### ISSUE TITLE
module 'arviz' has no attribute 'geweke'

### ISSUE THREAD

**Issue by keithbriggs** [pymc-devs/pymc#4503]
I did a "pip install pymc3", which reported a successful install.  But then:
```

kbriggs@Dell:~> python3
Python 3.6.9 (default, Jan 26 2021, 15:33:00) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pymc3 as pm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/pymc3/__init__.py", line 64, in <module>
    from .stats import *
  File "/usr/local/lib/python3.6/dist-packages/pymc3/stats/__init__.py", line 47, in <module>
    geweke = map_args(az.geweke)
AttributeError: module 'arviz' has no attribute 'geweke'
```

This is on an up-to-date Ubuntu 18 system.

**Comment by OriolAbril** [comment]
The solution is explained here: https://discourse.pymc.io/t/attributeerror-module-arviz-has-no-attribute-geweke/6818. I will close the issue as everything works fine if both latest arviz and pymc3 are installed.

**Comment by keithbriggs** [comment]
Ok, thanks.   But there is still a kind of bug if pip reports a successful install.   It must mean that the dependencies are not correctly specified somewhere.

**Comment by OriolAbril** [comment]
This happens with everything at some point. The root issue is that you have an old pymc3 version (and can't be updated because later versions no longer support python 3.6) but the latest arviz version that no longer supports that pymc3 version. When we released the pymc3 version you have installed we did not know when would arviz stop supporting it so it has no upper bound on the arviz version and we it can't be changed after releasing. You will run into the same problem when trying to use an old numpy version with a newer scipy version... (also note that as I also commented on Discourse, numpy, matplotlib, scipy... latest versions have also dropped py3.6 support)
